state/modeluser_test.go

```diff
@@ -45,7 +46,7 @@ func (s *ModelUserSuite) TestAddModelUser(c *gc.C) {
 
 	c.Assert(modelUser.UserID, gc.Equals, fmt.Sprintf("%s:validusername", s.modelTag.Id()))
 	c.Assert(modelUser.Object, gc.Equals, s.modelTag)
-	c.Assert(modelUser.UserName, gc.Equals, "validusername")
+	c.Assert(modelUser.UserName, gc.Equals, usertesting.GenNewName(c, "validusername"))
```

Comment:
These are all superfluous changes.
- Generating valid names is useful for input, but that should be validated in logic being tested. There's no need to run it through for result comparison.
- These packages are all disappearing.